README.rst

```diff
@@ -732,8 +732,11 @@ Responses as a context manager
         resp = requests.get('http://twitter.com/api/1/foobar')
         resp.status_code == 404
 
-Responses as a pytest fixture
------------------------------
+Using ``responses`` in unit tests
+---------------------------------
```

Comment:
Is this additional heading necessary? Responses is really only used in unit tests.

Reply:
@markstory 
we need some title to group. May be you can suggest something?

core idea is that this chapter describes integration with `pytest` and `unittest`

Reply:
@markstory 
maybe `integration with unit test frameworks` ?

Reply:
I did a rename